src/Microsoft.Sbom.Api/Output/ManifestToolJsonSerializer.cs

```diff
@@ -49,7 +49,7 @@ public async ValueTask DisposeAsync()
 
     /// <summary>
     /// This writes a json document to the underlying stream.
-    /// We also call dispose on the JsonDocument once we finish writing.
+    /// Disposal is taken care of in implementations of IJsonArrayGenerator after writing.
```

Comment:
Where it's freed does not matter, other than it's not freed by this code
```suggestion
    /// The jsonDocument object gets cleaned up by the caller, not here.
```